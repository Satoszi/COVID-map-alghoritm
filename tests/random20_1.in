20 20 6 78
random20_1
..+-+..+.+-.++.|+..|
+.|+.+..+.+|.+-.|+|+
-++.|...|+-..+.++++.
+--..-..+|-|+++..|++
.+.|.++++-.-+|..++|+
.|..++-+.+-++.+.++.|
+.-.|-.|.|....+..+.+
+-..+.+++++|.....-|+
.+..++|..|+|++..+..+
+...--....|...+....-
.+++..|+.-.+.+|.++|+
++..+++.+......+..|.
|.+++|+.++|--.+.|+..
++.|..+-++-.|+.+|++-
+.+.++|..+-+.+-.++.+
+..+++++|---.+++..|-
++...|.+.-..+.+.+...
++.++.-.|+++|++.+.++
+|..-+-+++|..+++..+.
.|+.+|++.+.|.++.+++.
