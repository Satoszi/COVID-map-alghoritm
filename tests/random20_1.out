11 19
4 19
2 19
13 18
4 18
1 18
18 17
11 17
6 17
3 17
16 16
12 16
9 16
19 15
18 15
9 15
8 15
19 14
11 14
4 14
2 14
15 13
13 13
12 13
1 13
16 12
14 12
11 12
10 12
2 12
15 11
19 10
18 10
16 10
13 10
11 10
9 10
7 10
3 10
19 9
14 9
10 9
4 9
0 9
19 8
16 8
12 8
6 8
5 8
1 8
18 7
17 7
11 7
5 6
2 6
0 6
14 5
11 4
3 4
1 4
11 3
10 3
5 3
0 3
9 2
4 2
11 1
10 1
8 1
5 1
3 1
0 1
19 0
15 0
12 0
9 0
7 0
2 0
